### ISSUE TITLE
Clear customizations after template export & adding to theme files

### ISSUE THREAD

**Issue by Humanify-nl** [WordPress/gutenberg#40432]
## What problem does this address?
My workflow with the site-editor currently is:
- Change the templates according to project requirements
- Export the templates from database and copy them in the theme (repo)
- Press clear customizations for 10+ template & parts manually.

## What is your proposed solution?
There needs to be some way for the editor to 'recognise' an import is done and that the theme files have been overwritten with the current (exactly similar) exported templates & parts that were imported in the theme files.

For now, what would help immensely is an option to 'clear all customizations', instead of having to press the buttons 20 times for each part.

Also this needs to clear automatically... because it is.. see for yourself:
![image](https://user-images.githubusercontent.com/82579622/163952568-7bcacfe2-52dd-4b88-bce8-9dd5924e8dd4.png)

**Comment by annezazu** [comment]
cc @scruffian as this seems in line with some work you've been doing around exports.

**Comment by scruffian** [comment]
Yes! Our suggested workflow is to allow you save the edits directly to the files as per: https://github.com/WordPress/gutenberg/issues/39194

**Comment by pbking** [comment]
@Humanify-nl Until this lands in core I've been using [this plugin](https://github.com/Automattic/create-block-theme) which has that very feature.  It [saves](https://github.com/Automattic/create-block-theme/pull/51) the templates and customizations in the active theme files and then clears those customizations from the user space.

Ultimately the most useful aspects of that plugin should make their way into Gutenberg and Core but this has helped my workflow quite a bit in the meantime.
